homeassistant/components/tailscale/strings.json

```diff
@@ -2,14 +2,14 @@
   "config": {
     "step": {
       "user": {
-        "description": "This integration monitors your Tailscale network, it **DOES NOT** make your Home Assistant accessible via Tailscale VPN. \n\nTo authenticate with Tailscale you'll need to create an API key at {authkeys_url}.\n\nA Tailnet is the name of your Tailscale network. You can find it in the top left corner in the Tailscale Admin Panel (beside the Tailscale logo).",
+        "description": "This integration monitors your Tailscale network, it **DOES NOT** make your Home Assistant accessible via Tailscale VPN. \n\nTo authenticate with Tailscale you'll need to create an API access token at {authkeys_url}.\n\nA Tailnet is the name of your Tailscale network. You can find it in the top left corner in the Tailscale Admin Panel (beside the Tailscale logo).",
```

Comment:
Yet the `api_key` is still the requested information. Should that not become `access token` too?

Reply:
According to their website, access token sometimes names as API key, but it gets confused with another credential (auth key) as the linked issue shows. Do you think should we keep `api_key` here?

Reply:
I suggest we keep it